### PR TITLE
fixed bold

### PR DESCRIPTION
**Pull request by cavukovic**
### Description

fixed bold so that only the word assigned is bold

### CODE REVIEW

File: packages/client/src/views/Dashboard.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@
                           <strong> interested </strong>
                       </span> in
                     </span>
-                    <span v-if="agencies.item.interested === 2" > <strong> was assigned </strong> </span>{{ agencies.item.grant }}
+                    <span v-if="agencies.item.interested === 2" >  was <strong> assigned </strong> </span>{{ agencies.item.grant }}
```
Comment (mhuang-usdr) [inline packages/client/src/views/Dashboard.vue:33]: Looks like there's a double space between was and assigned.

File: packages/client/src/views/RecentActivity.vue [hunk 2]
```diff
@@ -29,7 +29,7 @@
               <strong> interested </strong>
               </span> in
             </span>
-          <span v-if="agencies.item.interested === 2" > <strong> was assigned </strong> </span>{{ agencies.item.grant }}
+          <span v-if="agencies.item.interested === 2" > was <strong> assigned </strong> </span>{{ agencies.item.grant }}
```
Comment (mhuang-usdr) [inline packages/client/src/views/RecentActivity.vue:32]: Same - Looks like there's a double space between was and assigned.

**Review by mhuang-usdr — CHANGES_REQUESTED** [review]

File: packages/client/src/views/Dashboard.vue (hunk 1 above)
Comment (cavukovic) [reply packages/client/src/views/Dashboard.vue]: updated it

File: packages/client/src/views/RecentActivity.vue (hunk 2 above)
Comment (cavukovic) [reply packages/client/src/views/RecentActivity.vue]: updated it

**Review by mhuang-usdr — APPROVED** [review]

**Pull request merged**